utils/python/CIME/SystemTests/nck.py

```diff
@@ -107,6 +107,7 @@ def run(self):
         #======================================================================
 
         if success:
+            os.remove("%s/%s.exe" % (exeroot, cime_model))
```

Comment:
Can you explain what was happening that made this change necessary? Is it sometimes the case that the copy below fails, so you need to remove this file to ensure that the old executable isn't used?